rrfs-test/testinput_expr/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -62,27 +63,27 @@ cost function:
       weight:
         value: 0.1
 
-    - covariance:
+    - covariance: 
         covariance model: ensemble
         members from template:
           template:
             datetime: 2022-05-26T19:00:00Z
             filetype: fms restart
             state variables: *3dvars
-            datapath: Data/inputs/memMEM/
+            datapath: Data/inputs/mem%mem%/
             filename_core: 20220526.190000.fv_core.res.tile1.nc
             filename_trcr: 20220526.190000.fv_tracer.res.tile1.nc
             filename_sfcd: 20220526.190000.sfc_data.nc
             filename_sfcw: 20220526.190000.fv_srf_wnd.res.nc
             filename_cplr: 20220526.190000.coupler.res
-          pattern: MEM
-          nmembers: 20
+          pattern: %mem%
+          nmembers: 20 
           zero padding: 3
         localization:
           localization method: SABER
           saber central block:
             saber block name: BUMP_NICAS
-            active variables: *3dvars
+            active variables: *3dvars 
```

Comment:
@SamuelDegelia-NOAA  Would you please use this PR to add the below  (shown alongside with the existing "drivers.." block. 
```
         drivers:
         multivariate strategy: duplicated
         compute nicas: true

         model:
                level for 2d variables: last
``` to the fv3jedi var yaml?    As I mentioned in our meetings, this tell the bump to treat the 2d variables as the last level in a 3d structure data, which is needed since the fset is stored from top to bottom when passed to saber.  This option is not needed for this ctest since no 2d analysis variable is used like surface pressure. But adding it here would avoid possible future overlooing on this issue.

Reply:
Yes I can add that. Does this need to be added for just fv3-jedi or also mpas-jedi yamls?

Reply:
Thanks. I think mpas-jedi var yaml already have  it.

Reply:
The existing fv3jedi hybrid yaml has `read local nicas: true` instead of `compute nicas: true`. Did you also want to change that? Or just add the `level for 2d variables` under `model:`?

Reply:
Yes as we talked off-line. Just add the "level.." parameter. Thanks